Python/Product/EnvironmentsList/ToolWindow.xaml.cs

```diff
@@ -483,6 +481,12 @@ await StartRefreshDBAsync(view)
                     _condaEnvironmentView = null;
                 }
                 _onlineHelpView = EnvironmentView.CreateOnlineHelpEnvironmentView();
+
+                if (synchronous) {
+                    Dispatcher.Invoke(FirstUpdateEnvironments);
+                } else {
+                    Dispatcher.InvokeAsync(FirstUpdateEnvironments).Task.DoNotWait();
```

Comment:
Not a big deal, but the way I've approached this in other places is to keep the task around as an internal variable and wait on it in the test.